src/viam/components/board/board.py

```diff
@@ -38,11 +38,23 @@ class Analog:
         name: str
         """The name of the analog pin"""
 
+        class Value:
+            """
+            Data obtained from reading an analog pin. The value is the raw bits obtained, while the
+            range indicates indicates the what the extreme values indicate. The step_size is the
+            precision per bit of the reading. The value read, when interpreted in
+            volts/degrees/etc., is (min_range + value * step_size)
+            """
+            value: int
+            min_range: float
+            max_range: float
+            step_size: float
```

Comment:
Please put the individual attribute docs as docstrings. Then, the class docstring can be how to use this class

```python3
value: int
"""The raw bits obtained"""

min_range: float
"""The minimum of the range"""

max_range: float
"""..."""

step_size: float
"""Precision per bit of the reading"""
```